wpilibj/src/main/java/edu/wpi/first/wpilibj/util/Units.java

```diff
@@ -0,0 +1,94 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package edu.wpi.first.wpilibj.util;
+
+/**
+ * Utility class that converts between commonly used units in FRC.
+ */ 
+public class Units {
```

Comment:
Should be a final class with a private constructor that throws an UnsupportedOperationException:

```
  private Units() {
    throw new UnsupportedOperationException("This is a utility class!");
  }
```